src/__tests__/Network.ts

```diff
@@ -77,6 +77,18 @@ describe('Network Class', () => {
     });
   });
 
+  describe('method: getSs58Format', () => {
+    it("should return the chain's SS58 format", async () => {
+      const ss58Format = new BigNumber(42);
+
+      dsMockUtils.configureMocks({ contextOptions: { ss58Format } });
+
+      const result = await network.getSs58Format();
```

Comment:
```suggestion
      const result = network.getSs58Format();
```

Not async (sonar says its a smell)

Reply:
Oops. My bad. File remained unsaved. Pushed up a patch for it